src/phoenix/server/api/dataloaders/experiment_run_counts.py

```diff
@@ -27,14 +27,27 @@ def __init__(
 
     async def _load_fn(self, keys: List[Key]) -> List[Result]:
         experiment_ids = keys
+        resolved_experiment_ids = (
+            select(models.Experiment.id)
+            .where(models.Experiment.id.in_(set(experiment_ids)))
+            .subquery()
+        )
+        query = (
+            select(
+                resolved_experiment_ids.c.id,
+                func.count(models.ExperimentRun.experiment_id),
+            )
+            .select_from(resolved_experiment_ids)
+            .outerjoin(
+                models.ExperimentRun,
```

Comment:
an alternative
```suggestion
            .outerjoin_from(
                resolved_experiment_ids,
                models.ExperimentRun,
```

Reply:
addressed in 3b201a71c2c4f16656a14c86c0e5ca71d315a64e